tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "http://json.schemastore.org/tsconfig",
-	"extends": "@loopback/build/config/tsconfig.common.json",

```

Comment:
question: what happened here btw? did the inherited config stop working?

Reply:
Yeah it did not sure why